API-reference.md

```diff
@@ -281,6 +281,10 @@ Values that control how the telemetry data is sent.
         // The url from where the JS SDK will be downloaded. 
         // Default 'https://az416426.vo.msecnd.net/scripts/a/ai.0.js'
         url: string;
+
+        // If true, the SDK will not store or read any data from cookies.
+        // Default: false
+        disableCookies: boolean;
```

Comment:
can you move it closer to `cookieDomain` setting so they will be sitting together.